components/Task/Task.js

```diff
@@ -0,0 +1,71 @@
+import styled from "styled-components";
+import categoryColor from "@/utils/categoryColor";
+
+const StyledItem = styled.li`
+  max-width: 320px;
+  max-height: 400px;
+  border: 1px solid #cdc3b7;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  padding: 25px 15px;
+  background-color: rgba(255, 255, 255, 0.295);
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+  justify-content: space-between;
+  transition: all 0.5s;
+
+  &:hover {
+    background-color: rgba(0, 0, 0, 0.05);
+  }
+`;
+
+const StyledImageWrapper = styled.div`
+  width: 150px;
+  height: 150px;
+  overflow: hidden;
+  border-radius: 5px;
+`;
+
+const StyledImage = styled.img`
```

Comment:
Since you are working with Next.js, I recommend using the Image component from Next.

```suggestion
const StyledImage = styled(Image)`
```

Don't forget to adjust the next.config to whitelist the image domains.